packages/client/components/jobs/JobCard.tsx

```diff
@@ -1,138 +1,130 @@
 import {
   Box,
-  Container,
-  Link,
   Flex,
-  Spacer,
-  Image,
   Heading,
-  Text,
   HStack,
+  Image,
+  Link,
+  Text,
   Tag,
   TagLabel,
+  VStack,
 } from '@chakra-ui/react';
+import { Job } from '../../types/job'
+import { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHeart as faHeartSolid } from '@fortawesome/free-solid-svg-icons';
+import { faHeart as faHeartOpen } from '@fortawesome/free-regular-svg-icons';
+
+export default function JobCard(props: Job) {
+  // Defaults until we actually get some real data
+  // Probably not entirely necessary but will definitely be destructuring the data in some capacity.
+  const {
+    position = "Frontend Magician",
+    compensation = "0.6 ETH",
+    description = "We already have the best design skills in web3. No chance you’ll be able to improve on what we’ve got but why not come along and learn something?",
+  } = props;
+
+  const name = "Company Name",
+    organization_description = "NFT Marketplace",
+    logo_url = "/DevDAO.png",
+    technologyTags = ["Solidity", "EthersJS"];
+
+  const [isFavorited, setIsFavorited] = useState(true);
 
-export default function JobCard() {
   return (
     <Box
+      key={Math.floor(Math.random() * 1000)} // will use the actual job id
       boxSizing="border-box"
       boxShadow="0px 1px 1px rgba(0, 0, 0, 0.25), 1px 0px 1px rgba(0, 0, 0, 0.25)"
       height={{ lg: 'fit-content', md: 'fit-content', sm: 'max-content' }}
-      m="auto"
-      mb="0.5%"
-      mt="2.5%"
       w="100%"
+      mb="2.5%"
+      mr="2.5%"
       textAlign="left"
       bg="white"
+      minW="300px"
+      maxW="320px"
     >
-      <Box ml="0.5%" maxW="100%">
+      <Box p="25px" borderBottom="1px" borderColor="#F1F0F5">
+        <Flex align="center" justify="space-between">
+          <HStack>
+            <Link
+              href="/recruiter-name"
+              _hover={{ textDecoration: 'none' }}
+              _focus={{ textDecoration: 'none', border: 'none' }}
+            >
+              <Flex mt="1%" mb="2.5%">
+                <Image
+                  w="45px"
+                  h="45px"
+                  borderRadius="6px"
+                  border="1px solid black"
+                  src={logo_url}
+                  alt={`${name ?? "organization"} logo`}
+                  mr="15px"
+                />
+                <VStack spacing="5px" alignItems="start">
+                  <Heading size="sm">{name}</Heading>
+                  <Text fontSize="xs" color="#86848F">{organization_description}</Text>
+                </VStack>
+              </Flex>
+            </Link>
+          </HStack>
+          { isFavorited ?
+            <FontAwesomeIcon icon={faHeartSolid} color="#4E00EC" onClick={(() => setIsFavorited(false))} /> :
```

Comment:
I'm still working out a way to get a gradient on this, but I did use one of the purples from the gradient to at least keep it on brand in some capacity.